files/default/neutron-ha-tool.py

```diff
@@ -106,12 +106,12 @@ def parse_args():
                          'some reason.')
     wait_parser = ap.add_mutually_exclusive_group(required=False)
     wait_parser.add_argument('--wait-for-router', action='store_true',
-                    dest='wait_for_router')
+                             dest='wait_for_router')
```

Comment:
flake8 is separate to unit testing so should have been in a separate commit.  I was initially confused by trying to figure out how these whitespace changes were related to unit testing, before realising that they are not.